_posts/2016-01-22-let-vs-instance.markdown

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: "Let vs Instance Variables"
+date: 2016-01-22 10:36:00
+categories: ["rails", "rspec"]
```

Comment:
@schmierkov I think we could add `ruby` as well, as any pure-Ruby project can use the information in this post too.